File: packages/pensopay_android/android/src/main/kotlin/com/flutter/pensopay/networking/pensopayapi/pensopaylink/models/PPBasket.kt
package com.flutter.pensopay.networking.pensopayapi.pensopaylink.models

import org.json.JSONObject

class PPBasket(qty: Int,
               item_no: String,
               item_name: String,
               item_price: Double,
               vat_rate: Double): JSONObject() {

    // Required properties

    var qty: Int = qty
    var item_no: String = item_no
    var item_name: String = item_name
    var item_price: Double = item_price
    var vat_rate: Double = vat_rate

}

File: packages/pensopay_ios/ios/Classes/Networking/Pensopay API/Pensopay Link/Models/PPBasket.swift
//
//  PPBasket.swift
//  PensopaySDK
//
//  Created on 07/26/2022
//  Copyright © 2022 Pensopay. All rights reserved.
//

import Foundation

public class PPBasket : Codable {

    // MARK: - Properties
    
    public var qty: Int
    public var item_no: String
    public var item_name: String
    public var item_price: Double
    public var vat_rate: Double
 
    
    // MARK: Init
    
    public init(qty: Int, item_no: String, item_name: String, item_price: Double, vat_rate: Double) {
        self.qty = qty
        self.item_no = item_no
        self.item_name = item_name
        self.item_price = item_price
        self.vat_rate = vat_rate
    }
    
}
